docs/sdk/quickstart-by-language/quickstart-nodejs.mdx

```diff
@@ -0,0 +1,156 @@
+This tutorial is for users who have already set up at least one policy in Permit.io. If you haven't done so yet,
+please refer to [this guide](/quickstart) first.
+:::
+
+## Adding the Permit.io Node SDK to your project
+
+### 1. Install the Permit.io SDK
+
+To start, you need to install the Permit.io SDK in your project. Open your terminal and run the following command:
+
+```sh
+npm install permitio
+```
+
+This will add the Permit.io SDK to your project, allowing you to integrate authorization management seamlessly.
+
+### 2. Import the SDK into your code
+
+After installing the SDK, you need to import it into your application.
+
+### 3. Create a new Instance of the SDK
+
+Next, create a new instance of the Permit SDK. You'll need a secret API key, which you can obtain by following the instructions
+provided in the setup section of the Permit.io documentation.
+
+```javascript
+const permit = new Permit({
+  token: "[YOUR_API_KEY]",
+  pdp: "http://localhost:7766",
+});
+```
+
+Replace "YOUR_API_KEY" with the actual API key you obtained. Find instructions how to fetch your API key here.
+
+## The fundamental functionality - Checking for permissions
+
+`permit.check()` is the fundamental SDK function that controls the enforcement in your application level code. It acts as a guard of who
+can do what, based on the policy that you defined.
+
+Each user, once his identity is verified by the authentication provider, gets issued a session. During this session, his unique id
+obtained from the JWT (JSON Web Token), which is synced to Permit, maps to a specific role within the Policy.
+
+Each role has a set of rules that were pre-defined, which a policy engine can understand, evaluate, and therefore return if the user
+is permitted or not, based on the user id, action and resource passed into the SDK function.
+
+You can run a permission check with permit.check(), passing in 3 arguments:
+
+- `user`: a unique string id that identifies the user doing the action (and the currently logged in user).
+- `action`: the action performed.
+- `resource`: the resource (object) the action is performed on.
+
+In the following example we are checking that a user with the unique id `john@smith.com` can `read` a `document`.
+
+```javascript
+const permitted = await permit.check("john@permit.io", "read", "document");
+
+if (permitted) {
+  console.log("John is PERMITTED to read a document");
+} else {
+  console.log("John is NOT PERMITTED to read a document");
+}
+```
+
+## Other basic functionality with the SDK
+
+In this quickstart, we'll introduce you to some core Permit.io functionality using the SDK. As you'll see, the SDK is designed to
+be simple and logical, making it easy to develop and manage policies.
+
+### Creating a resource
+
+A resource is the fundamental building block of a policy. It's what the user can interact with. It could be a 'document', a 'building'
+or even a 'patient health record'.
+
+```javascript
+await permit.api.resource.create({
```

Comment:
In this case, I would add the resource instance action before the resource itself. The reason is the cadence of using such API calls via the API. We should focus more on the `data` operations and less on the `control` operations. We could have one for each plan, starting from the data and moving up.
- Sync user (very important and relevant)
- Role assignment
- Resource creation
- Copy environment

Reply:
Actually, this answered by above question